models/comments.model.js

```diff
@@ -26,6 +26,7 @@ exports.insertCommentByArticleId = async (articleId, newComment) => {
 }
 
 exports.removeCommentById = async (commentId) => {
+    await checkResourceExists('comments', 'comment_id', commentId);
```

Comment:
Shouldn;t need to check if the comment exists here - you can take a look at the rows property brought back by the delete query - if empty, it never existed in the first place